test/videojs-contrib-hls.test.js

```diff
@@ -2695,6 +2711,9 @@ QUnit.module('HLS Integration', {
     this.tech = new (videojs.getTech('Html5'))({});
     this.clock = this.env.clock;
 
+    this.originalSelectPlaylist = Hls.STANDARD_PLAYLIST_SELECTOR;
+    Hls.STANDARD_PLAYLIST_SELECTOR = movingAverageBandwidthSelector();
```

Comment:
Yes, but because of the way the closure works, if we don't do this between tests, each test will be using the moving average of all the previous tests